nixos/modules/services/monitoring/prometheus/alertmanager.nix

```
@@ -119,26 +136,34 @@ in {
     };
   };
 
+  config = mkMerge [
+    (mkIf cfg.enable {
+      assertions = singleton {
+        assertion = cfg.enable && (cfg.configuration != null || cfg.configText != null);
```

I think this assertion for `cfg.enable` can be dropped because it seems to be redundant after the `mkIf` above, but otherwise the changes look good to me :+1: 
```suggestion
        assertion = (cfg.configuration != null || cfg.configText != null);
```